libafl/src/corpus/testcase.rs

```diff
@@ -432,6 +469,21 @@ impl SchedulerTestcaseMetadata {
 
 crate::impl_serdeany!(SchedulerTestcaseMetadata);
 
+#[cfg(feature = "std")]
+impl<I> Drop for Testcase<I>
+where
+    I: Input,
+{
+    fn drop(&mut self) {
+        if let Some(filename) = &self.filename {
+            let mut path = PathBuf::from(filename);
```

Comment:
I am not sure if every Testcase for every corpus needs to have a file, just because it has a filename(?)

Reply:
Not super clear on what you mean here. Won't testcase filenames only be generated iff the testcase is written to disk?